10up/insecure-content-warning
# Move translations into our JS files

### Description of the Change

We currently translate all strings (besides one) in PHP and then load those into the JS via `wp_localize_script`. I'm guessing this is because all of this was added prior to their being an easier way to translate strings directly on the JS side.

This PR removes all the translations in our `wp_localize_script` call and adds those directly to our JS files, taking advantage of the various `wp.i18n` functions. For the most part, this is a one-to-one match of what we already have, just moving the translations into a different place, but we are now handling the plural translations better, using the `_nx` function.

### Alternate Designs

None

### Benefits

Translations are handled directly where the text is being used, which is cleaner. Also properly translating our plural string

### Possible Drawbacks

None

### Verification Process

1. Open a post in Gutenberg that has insecure elements
2. Try to save the post
3. Note the warning messages that are shown. All text should match what was there previously

Run same test in the Classic Editor

### Checklist:


### Applicable Issues

Fixes #23
